lib/engine/game/g_1824/step/buy_sell_par_exchange_shares.rb

```diff
@@ -161,6 +163,15 @@ def invalid_mountain_railway_exchange?(entity, corporation, exchange)
 
             !@game.mountain_railway_exchangable? || !@game.exchangable_for_mountain_railway?(entity, corporation)
           end
+
+          def add_pass_to_allow_mr_change?(entity, actions)
+            # There might be that we have no other possible actions, but still have an MR we might want to exchange
+            return false unless entity.player?
+            return false unless @game.mountain_railway_exchangable?
+            return false if bought? || actions.include?('pass')
+
+            @game.companies.find { |c| @game.mountain_railway?(c) && c.owner == entity }
```

Comment:
Should we check for closed as well, just to be sure that closed MRs does not affect this?